src/file-1.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:37913/organizations/org5/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:37913/project/issues?id=sample&pullRequest=6978&issues=AWlc6RT3Qm3x9CvtX8o2&open=AWlc6RT3Qm3x9CvtX8o2)

[ISSUE_KEY]: AWlc6RT3Qm3x9CvtX8o2